tests/test_monitor.py

```diff
@@ -37,22 +21,19 @@ def test_response_values(self, r):
             assert isinstance(response['client_port'], unicode)
             assert response['command'] == 'PING'
 
-    @skip_if_server_version_lt('5.0.0')
```

Comment:
Why do we have this in the first place? Redis has`monitor` since 1.0 https://redis.io/commands/monitor @andymccurdy

Reply:
The `wait_for_command()` function previously relied on the [`CLIENT ID`](https://redis.io/commands/client-id) command, added in Redis 5.0.0.

Reply:
Oh I see, very nice fix!